media-libs/alembic/alembic-1.7.7.recipe

```diff
@@ -67,21 +90,25 @@ BUILD_PREREQUIRES="
 
 defineDebugInfoPackage alembic$secondaryArchSuffix \
 	$libDir/libAlembic.so.$portVersion \
-	$binDir/abcconvert \
-	$binDir/abcdiff \
-	$binDir/abcecho \
-	$binDir/abcechobounds \
-	$binDir/abcls \
-	$binDir/abcstitcher \
-	$binDir/abctree
+	$commandBinDir/abcconvert \
+	$commandBinDir/abcdiff \
+	$commandBinDir/abcecho \
+	$commandBinDir/abcechobounds \
+	$commandBinDir/abcls \
+	$commandBinDir/abcstitcher \
+	$commandBinDir/abctree
 
 BUILD()
 {
 	mkdir -p build
 	cd build
+
+	export ILMBASE_INC_DIR="`finddir B_SYSTEM_DIRECTORY`/$relativeIncludeDir/OpenEXR"
+
 	cmake .. \
 		-DUSE_HDF5=ON \
 		-DALEMBIC_LIB_USES_BOOST=ON \
+		-DLIBRARY_INSTALL_DIR="$relativeLibDir" \
```

Comment:
We really need to drop this line:
```diff
-		-DLIBRARY_INSTALL_DIR="$relativeLibDir" \
```
and, instead, add the `PATCH()` function. Otherwise `INSTALL()` fails with this error:
```
mv: cannot stat '/packages/alembic_x86-1.7.7-1/.self/lib/x86/libAlembic.*': No such file or directory
```
Here's the `PATCH()` function that will make `INSTALL()` happy:
```bash
PATCH()
{
	if [ -n "$secondaryArchSuffix" ]; then
		sed -i \
			-e "s|\(DESTINATION\) lib|\1 $relativeLibDir|;" \
			-e "s|\(ConfigPackageLocation\) lib/cmake/|\1 $relativeLibDir/cmake/|;" \
			lib/Alembic/CMakeLists.txt
	fi
}
```